prExplorer/API/obj/API.csproj.nuget.dgspec.json

```diff
@@ -1,17 +1,17 @@
 {
   "format": 1,
   "restore": {
-    "c:\\Users\\Ermat\\Desktop\\NC Github\\NC\\prExplorer\\API\\API.csproj": {}
+    "C:\\Users\\Ermat\\Desktop\\NC Github\\NC\\prExplorer\\API\\API.csproj": {}
```

Comment:
You should not check in those files to the repo ;)

Reply:
@aliyorulmazdev I see that you added `.gitignore` file on the `main` branch. However, as all these files were already checked in, you need to remove them now manually